src/EnergyPlus/CondenserLoopTowers.hh

```diff
@@ -247,9 +247,6 @@ namespace CondenserLoopTowers {
         Real64 DesInletWaterTemp = 0.0;          // design tower inlet water temperature (C)
         Real64 DesOutletWaterTemp = 0.0;         // design tower outlet water temperature (C)
         Real64 DesInletAirDBTemp = 0.0;          // design tower inlet air dry-bulb temperature (C)
-        Real64 DesInletAirWBTemp = 0.0;          // design tower outlet air wet-bulb temperature (C)
-        Real64 DesApproach = 0.0;                // design tower approach temperature (deltaC)
-        Real64 DesRange = 0.0;                   // design tower range temperature (deltaC)
```

Comment:
Remove duplicated Variables on CoolingTower struct. I kept only the ones defined first in the file.

| Kept           | Removed           |
|----------------|-------------------|
| DesignInletWB  | DesInletAirWBTemp |
| DesignApproach | DesApproach       |
| DesignRange    | DesRange          |

Reply:
https://github.com/NREL/EnergyPlus/blob/31e3c33467c5873371bf48b12a7318215971c315/src/EnergyPlus/CondenserLoopTowers.hh#L165-L167

Reply:
The report was zero for "Cooling Tower Design Inlet Wet-Bulb Temperature" because of the fact that two variables exited for the same quantity.

DesignInletWB was the one that stored the CT:VariableSpeed input: 

https://github.com/NREL/EnergyPlus/blob/31e3c33467c5873371bf48b12a7318215971c315/src/EnergyPlus/CondenserLoopTowers.cc#L1302

But the other one was reported

https://github.com/NREL/EnergyPlus/blob/31e3c33467c5873371bf48b12a7318215971c315/src/EnergyPlus/CondenserLoopTowers.cc#L3379